packages/contracts-bedrock/contracts/echidna/FuzzOptimismPortal.sol

```diff
@@ -3,16 +3,37 @@ pragma solidity 0.8.15;
 import { OptimismPortal } from "../L1/OptimismPortal.sol";
 import { L2OutputOracle } from "../L1/L2OutputOracle.sol";
 import { AddressAliasHelper } from "../vendor/AddressAliasHelper.sol";
+import { SystemConfig } from "../L1/SystemConfig.sol";
 
 contract EchidnaFuzzOptimismPortal {
     OptimismPortal internal portal;
     bool internal failedToComplete;
 
     constructor() {
+        SystemConfig.ResourceConfig memory config = SystemConfig.ResourceConfig({
+            maxResourceLimit: 20_000_000,
+            elasticityMultiplier: 10,
+            baseFeeMaxChangeDenominator: 8,
+            minimumBaseFee: 1 gwei,
+            systemTxMaxGas: 1_000_000,
+            maximumBaseFee: type(uint128).max
+        });
```

Comment:
Do we still have a constants.sol (and a constants.ts) file that this could be the duplicated in?

Reply:
moved to constants, good call